pkg/network/tracer/tracer_classification_test.go

```diff
@@ -157,6 +165,197 @@ func testProtocolClassification(t *testing.T, cfg *config.Config, clientHost, ta
 	}
 }
 
+func testKafkaProtocolClassification(t *testing.T, cfg *config.Config, clientHost, targetHost, serverHost string) {
+	const topicName = "franz-kafka"
+	testIndex := 0
+	// Kafka does not allow us to delete topic, but to mark them for deletion, so we have to generate a unique topic
```

Comment:
So why are we deleting the topic in the teardown phase?

Reply:
to ensure no test will re-use the same topic